app/controllers/api/v1/accounts/statuses_controller.rb

```diff
@@ -9,7 +9,8 @@ class Api::V1::Accounts::StatusesController < Api::BaseController
   def index
     cache_if_unauthenticated!
     @statuses = load_statuses
-    render json: @statuses, each_serializer: REST::StatusSerializer, relationships: StatusRelationshipsPresenter.new(@statuses, current_user&.account_id)
+    return_source = params[:client_type] == 'editor'
```

Comment:
Minor naming/style things...

- Instead of a local var, move this to a `return_source?` query method elsewhere in controller, and use that to pass bool value to serializer
- I'd probably name this after the desired additional output, not where the request is being made from. Something like `params[:include_original] == true` or something.
- Exception to that would be if we foresee *many* other fields also being required only for editors
- Another avenue is that we do have 1 or 2 places where we pass in the desired fields to a serializer. Could do some sort of graphql-style ability here to ask for specific fields (from an allowed list) in a more general way for clients.